modules/outconBidAdapter.js

```diff
@@ -0,0 +1,41 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {config} from '../src/config';
+const BIDDER_CODE = 'outcom';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['outcom'],
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.pod || (bid.params.internalId && bid.params.publisher));
+  },
+
+  buildRequests: function(validBidRequests) {
+    let par = '';
+    if (validBidRequests.params.pod != undefined) par = 'get?pod=' + validBidRequests.params.pod;
```

Comment:
The `validBidRequests` is an array of potential bid requests to your adapter (in case the publisher has you listed in multiple adUnit objects on their page).

Can you please update the internal logic in this function to access the array?  It's currently throwing undefined errors.